### PR TITLE
add sentry-sdk modules

### PR DESCRIPTION
**Pull request by chssn**

### CODE REVIEW

File: cx_Freeze/hooks.py [hunk 1]
```diff
@@ -77,6 +77,7 @@ def initialize(finder: ModuleFinder) -> None:
     finder.ExcludeModule("Queue")
     finder.ExcludeModule("rexec")
     finder.ExcludeModule("robotparser")
+    finder.ExcluedModule("sentry_sdk")
```
Comment (marcelotduarte) [inline cx_Freeze/hooks.py]: Is this entry required? Usually, only load_sentry will be needed. Entries in this section are used to delete old modules like the ones that changed their name in the transition from py2 to py3.

File: cx_Freeze/hooks.py [hunk 2]
```diff
@@ -1157,7 +1158,21 @@ def load_reportlab(finder: ModuleFinder, module: Module) -> None:
     """
     finder.IncludeModule("reportlab.rl_settings")
 
-
+    
+def load_sentry(finder: ModuleFinder, module: Module) -> None:
+    """
+    The Sentry.io SDK
+    """
+    find.IncludeModule("sentry_sdk.integrations.stdlib")
```
Comment (marcelotduarte) [inline cx_Freeze/hooks.py]: typo (finder is the correct).

**Review by marcelotduarte — CHANGES_REQUESTED** [review]

File: cx_Freeze/hooks.py (hunk 1 above)
Comment (chssn) [reply cx_Freeze/hooks.py]: Probably not. I was trying to port across some code from PyInstaller to get the sentry_sdk working. Happy to go with those changes.

**Review by marcelotduarte — APPROVED** [review]

**Comment by marcelotduarte** [comment]
Thanks!

**Pull request merged**